evap/evaluation/templatetags/evaluation_filters.py

```diff
@@ -112,27 +116,29 @@ def weight_info(evaluation):
 def statename(state):
     return STATE_NAMES.get(state)
 
+@register.filter
+def state_from_english(state_str):
+    return STR_TO_STATE[state_str]
 
 @register.filter
 def statedescription(state):
     return STATE_DESCRIPTIONS.get(state)
 
-
 @register.filter
 def approval_state_values(state):
     if state in APPROVAL_STATES:
         return APPROVAL_STATES[state]
-    if state in ['in_evaluation', 'evaluated', 'reviewed', 'published']:
-        return APPROVAL_STATES['approved']
+    if state > Evaluation.State.APPROVED:
+        return APPROVAL_STATES[Evaluation.State.APPROVED]
     return None
 
 
 @register.filter
 def approval_state_icon(state):
     if state in APPROVAL_STATES:
         return APPROVAL_STATES[state].icon
-    if state in ['in_evaluation', 'evaluated', 'reviewed', 'published']:
-        return APPROVAL_STATES['approved'].icon
+    if state > Evaluation.State.APPROVED:
```

Comment:
Should this go back to checking in the list, or is it fair to assume that `state' is actually a valid state number?

Reply:
i think that's fair to assume after you've verified every caller ;)